package Message; 
message CreatePlayer 								//创建角色
{ 
   optional int64     uid 		= 1;  //账号ID
   optional int32			server	= 2;	//选择的区服号
   optional string    name 		= 3;  //角色名
   optional int32			roletemplate= 4;	//模板ID
   optional int64			pid			= 5;	//角色流水号
   optional	int32			stunt		= 6;	//绝技
   optional	int32			quality = 7;	//品质
} 